cmd/neofs-adm/internal/modules/morph/initialize.go

```diff
@@ -320,21 +325,28 @@ func (c *clientContext) awaitTx(cmd *cobra.Command) error {
 		}
 	}
 
+	err := awaitTx(cmd, c.Client, c.Hashes)
+	c.Hashes = c.Hashes[:0]
+
+	return err
+}
+
+func awaitTx(cmd *cobra.Command, c Client, hashes []util.Uint256) error {
 	cmd.Println("Waiting for transactions to persist...")
 
-	tick := time.NewTicker(c.PollInterval)
+	tick := time.NewTicker(time.Second)
 	defer tick.Stop()
 
-	timer := time.NewTimer(c.WaitDuration)
+	timer := time.NewTimer(30 * time.Second)
```

Comment:
Could you use a (local) constant here with a link to #1741 ?

Reply:
done